src/main/java/org/broadinstitute/consent/http/health/SendGridHealthCheck.java

```diff
@@ -0,0 +1,68 @@
+package org.broadinstitute.consent.http.health;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.google.api.client.http.HttpStatusCodes;
+import com.google.gson.Gson;
+import io.dropwizard.lifecycle.Managed;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.broadinstitute.consent.http.configurations.MailConfiguration;
+import org.broadinstitute.consent.http.util.HttpClientUtil;
+
+import java.nio.charset.Charset;
+
+public class SendGridHealthCheck extends HealthCheck implements Managed {
+    private final HttpClientUtil clientUtil;
+    private final MailConfiguration configuration;
+
+    public SendGridHealthCheck(HttpClientUtil clientUtil, MailConfiguration configuration) {
+        this.clientUtil = clientUtil;
+        this.configuration = configuration;
+    }
+
+    @Override
+    protected Result check() throws Exception {
+        try {
+            String statusUrl = configuration.getSendGridStatusUrl();
+            HttpGet httpGet = new HttpGet(statusUrl);
+            try (CloseableHttpResponse response = clientUtil.getHttpResponse(httpGet)) {
+                if (response.getStatusLine().getStatusCode() == HttpStatusCodes.STATUS_CODE_OK) {
+                    String content = IOUtils.toString(response.getEntity().getContent(), Charset.defaultCharset());
+                    SendGridStatus sgStatus = new Gson().fromJson(content, SendGridStatus.class);
+                    if (sgStatus.status.description.equalsIgnoreCase("All Systems Operational")) {
```

Comment:
Some points to consider
1. Since we now know more about what constitutes a healthy status (see https://status.sendgrid.com/api#status), we can be more accurate about what we're checking. This should be looking at the indicator property for one of these values `none, minor, major, or critical`. 
2. You have defined an object, `SendGridStatus`, that should represent the complete response from Sendgrid. We should flesh that out some more for completeness.
3. The logic of `Am I healthy?` really belongs inside the object itself. We should be parsing a `SendGridStatus` and then getting the boolean healthy value from that object. It's just a calculation once the full object is parsed.